BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/bank/bunq/BunqBank.java

```diff
@@ -162,6 +161,25 @@ private Void loadOrCreateClientKeys() {
         });
     }
 
+    @Override
+    public CompletableFuture<Boolean> sendTransaction(Transaction transaction) {
+        BunqTransaction tran = (BunqTransaction) transaction;
```

Comment:
instance of is noet needed here since transaction is extended by bunqtransaction so this will always succeed